adapters/repos/db/vector/hnsw/vector_cache.go

```diff
@@ -36,6 +37,7 @@ func newCache(getFromSource VectorForID, logger logrus.FieldLogger) *vectorCache
 		count:         0,
 		maxSize:       50000, // TODO: make configurable
```

Comment:
Unrelated to the review, but I'm just noticing this TODO, which is a reminder that we probably need one more issue to make all kinds of hard-coded values configurable. I'll open an issue where we can discuss the APIs for that. We might also take that into the Modularization milestone, as we also have to add the per-module config.

Reply:
Ok!